.gitignore

```diff
@@ -73,3 +73,6 @@ typings/
 
 # Mac files
 .DS_Store
+
+# IntelliJ IDE
+.idea
```

Comment:
I would recommend [adding this to your local global `.gitignore`](https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/), as we don't have any IDE-specific ignore rules at the moment.

Reply:
ah ok. Will update. I live in the world of monster .gitignore like this one: https://github.com/facebook/react-native/blob/main/.gitignore